dataactbroker/handlers/aws/session.py

```diff
@@ -198,18 +183,27 @@ def open_session(self, app, request):
         app -- (Flask) the Flask applcation
         request -- (Request)  the request object
 
-        implements the open_session method that pulls or creates a new DynamoSession object
+        implements the open_session method that pulls or creates a new UserSession object
 
         """
         sid = request.headers.get("x-session-id")
         if(sid and SessionTable.doesSessionExist(sid)):
             if SessionTable.getTimeout(sid)> toUnixTime(datetime.utcnow()):
-                return DynamoSession(initial=SessionTable.getData(sid),sid=sid)
+                session_dict =  UserSession()
+                # Replace single quotes with double and read as dict, also need to correct True, False, and None for json
+                data_string = SessionTable.getData(sid).replace("'",'"').replace("True","true")\
+                    .replace("False","false").replace("None","null")
+                data = loads(data_string)
+                for key in data.keys():
+                    session_dict[key] = data[key]
+                session_dict["sid"] = sid
```

Comment:
This line shouldn't be doing anything, right? It should have be in the serialized data.